api/apps/api/src/modules/projects/projects.controller.ts

```diff
@@ -66,8 +70,10 @@ export class ProjectsController {
   constructor(
     private readonly projectsService: ProjectsService,
     private readonly geoFeatureSerializer: GeoFeatureSerializer,
+    private readonly geoFeatureService: GeoFeaturesService,
     private readonly projectSerializer: ProjectSerializer,
     private readonly jobsStatusSerizalizer: JobStatusSerializer,
+    private readonly shapefileService: ShapefileService,
```

Comment:
It should not be used here.

It's a service from the geo app, and we should not share anything between the app `as is` but by shared libs.
The second thing is the service uses mapshaper under the hood, and we wanted to keep heavy calculation on the geo side – with this, we're breaking the rule.